bucket/ludusavi.json

```diff
@@ -0,0 +1,33 @@
+{
+    "version": "0.3.0",
+    "description": "Ludusavi is a tool for backing up your PC video game save data, written in Rust",
+    "homepage": "https://github.com/mtkennerly/ludusavi",
+    "license": {
+        "identifier": "MIT",
+        "url": "https://github.com/mtkennerly/ludusavi/blob/master/LICENSE"
+    },
```

Comment:
```suggestion
    "license": "MIT",
    "suggest": {
        "vcredist": "vcredist2019"
    },
```

Reply:
Regarding vcredist2019, the application should be linked statically to the runtime so that you actually don't need to install the redistributable. Did your testing find that it's actually required in spite of that?

Reply:
https://github.com/mtkennerly/ludusavi/releases/tag/v0.3.0 > https://github.com/mtkennerly/ludusavi/commit/0a6874fc396aa23dfc5669d0f007eea07906a373 > README #L39

![image](https://user-images.githubusercontent.com/13260377/87408291-90642c00-c5c3-11ea-9e19-6c04f21a7e41.png)

If you static link it now, then it could be removed.

Reply:
Yes, https://github.com/mtkennerly/ludusavi/commit/0a6874fc396aa23dfc5669d0f007eea07906a373 was followed up by https://github.com/mtkennerly/ludusavi/commit/a7d93fdcfbd5a9e475aeec8ba5bff54f4bf761ae , which enabled static linking and removed that note from the README. That commit was included in the v0.3.0 release ( https://github.com/mtkennerly/ludusavi/commit/edf6fadc1d7a0dca41e5e258809411f18d08d20b ).